themes/lcars.yaml

```diff
@@ -660,7 +660,7 @@
         margin-left: -16px;
       }
     }
-    @media screen and (max-width: 375px) {
+    /*@media screen and (max-width: 375px) {
```

Comment:
I meant to remove this portion before release. It was an experimental fix for #38 before I figured out the actual cause. We can just remove this whole commented portion.